src/machine.rs

```diff
@@ -270,6 +270,38 @@ impl<'m> Machine<'m> {
         Ok(())
     }
 
+    /// Delete the machine.
+    ///
+    /// Deletes the machine, cleaning up all associated resources.
+    ///
+    /// Machine is shutdown before resources are deleted.
```

Comment:
As per my other suggestion:

```suggestion
    /// Machine is shutdown (if its running) before resources are deleted.
```